### ISSUE TITLE
Accessibility: MAS20: alt text  is empty for the image links.

### ISSUE THREAD

**Issue by wiproaccess** [wiproaccess/vscode-website#4]
#A11yMAS
#MAS20 
#Wipro
#vscode-website

**Testing Environment:**
OS-RS1
OS Name Microsoft -Windows 10 Pro
Browser-Edge
Edge Version-Microsoft Edge 38.14393.461.0

**Repro Steps:**
1.Open https://code.visualstudio.com/docs/languages/overview 
2.Press F12 key and check the alt text for the image links under the "Programming Languages" below the header of the page.[screenshot1]

Note: Same issue is observed in below link
https://code.visualstudio.com/docs/runtimes/nodejs
[screenshot2]

**Actual Result:**
The alt text is empty for the images under "Programming Languages" below the header of the page.

**Expected Result:**
There should be alt text for the image links under "Programming Languages" below the header of the page.

**User Impact:**
Users with memory and attention limitations benefit by being able to quickly rediscover their context after an interruption.

![2_alt is empty for image links npm](https://cloud.githubusercontent.com/assets/24825441/21672312/bea1cdec-d347-11e6-8f15-e8b8c9656636.png)
![1_alt is empty for image links](https://cloud.githubusercontent.com/assets/24825441/21672318/c63890ea-d347-11e6-99ba-a325343de1c0.png)

**Comment by waderyan** [comment]
Fixed 
![image](https://cloud.githubusercontent.com/assets/2449568/26379011/e1a469da-3fcb-11e7-9da9-f9eb17484f1b.png)
